tests/bluemira/test_data/reactors/SMOKE-TEST-EU-DEMO/systems_code/mockPROCESS.json

```diff
@@ -27,7 +27,6 @@
     "g_vv_ts": 0.12,
     "h_cp_top": 6.0,
     "h_tf_max_in": 6.5,
-    "h_tf_min_in": -6.5,
```

Comment:
I've removed this (and below) because there isnt a process mapping for them (or I can't find one)

Reply:
Think you might need to remove them in the EUDEMO example as well, I get this when running the example:

![image](https://user-images.githubusercontent.com/26097289/159259639-c857046b-3e05-4cd5-97eb-18e4ea837b17.png)

Reply:
Done